src/commcare_cloud/ansible/roles/commcarehq/templates/supervisor_celery_beat.conf.j2

```diff
@@ -1,4 +1,4 @@
-[program:{{ project }}-{{ deploy_env }}-celerybeat]
+[program:{{ project }}-{{ deploy_env }}-celery_beat_0]
```

Comment:
Do you also want to change the log names? I'm not sure if there are any other automated tasks that would break for them

Reply:
https://github.com/dimagi/commcare-cloud/pull/2039/commits/8ba2051711e33c47529c3205ebb17a403dc3ddfd
That was the only place I could find `celerybeat` in the codebase